sixtyfps_runtime/corelib/items.rs

```diff
@@ -57,6 +57,22 @@ type ItemRendererRef<'a> = &'a mut dyn crate::item_rendering::ItemRenderer;
 pub type VoidArg = ();
 type KeyEventArg = (KeyEvent,);
 
+#[macro_export]
+macro_rules! declare_item_vtable {
+    (fn $getter:ident() -> $item_vtable_ty:ident for $item_ty:ty) => {
```

Comment:
Maybe with concat! and  #[link_name] it is possible to make this macro not take a $getter.

For example
```rust
#[link_name = concat!("sixtyfps_get_", stringify!($item_vtable_ty))]
fn $item_vtable_ty(){ ... }
```

But i think that's probably not worth it, and the way it is now is fine.

I actually like this syntax with `fn` and a return value.

Reply:
Ohh yes that’s better. I’ll give it a try, it might improve it.